Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM docker.io/golang:1.23-alpine3.20 AS builder
```

Comment:
1.23 uses latest golang within 1.23 family, the image is updated in docker registry on each patch release an right now points to 1.23.6

https://hub.docker.com/_/golang